lib/Basics/VelocyPackHelper.cpp

```diff
@@ -911,6 +911,7 @@ void VelocyPackHelper::patchDouble(VPackSlice slice, double value) {
   }
 #else
   // other platforms support unaligned writes
+  // cppcheck-suppress *
   *reinterpret_cast<double*>(p + 1) = value;
```

Comment:
Shouldn‘t this use memcpy?

Reply:
This place is intentionally *not* using memcpy.
It is non-portable, but guard by an `#ifdef` that tells whether unaligned memory access and the like is "acceptable".
I again verified this with compiler explorer and it seems that neither g+++6.3 nor g++8.2 do optimize the loop away to a simple store.